data/helpers/recipeDb.js

```diff
@@ -0,0 +1,19 @@
+const db = require("../dbConfig.js");
+
+module.exports = {
+  getDishes: function() {
+    return db("dishes");
+  },
+
+  addDish: function(dish) {
+    return db("dishes")
+      .insert(dish)
+      .then(ids => ({ id: ids[0] }));
+  },
+
+  getDish: function(id) {},
```

Comment:
It seems You are missing three methods here, did You have trouble implementing them?